src/scenes/Esittelijat/UusiAsiaEsidialog.js

```diff
@@ -11,7 +11,7 @@ import {
   Button
 } from "@material-ui/core";
 import { useOrganisations } from "../../stores/organisations";
-import { map, prop, sortBy } from "ramda";
+import * as R from "ramda";
```

Comment:
Why do you like to import everything from Ramda? I'm not delighted about the change but it doesn't prevent approving this PR.

Reply:
i thought code would be more readable when ramda functions are on their own namespace. side effect is that everything is imported and tree shaking and other optimizations cannot be run as effectively as before.

Maybe i will revert these changes